charts/cadvisor/values.yaml

```diff
@@ -47,6 +47,9 @@ podAnnotations: {}
 # sometimes errors are encountered when using the cpu load reader without being on the host network
 hostNetwork: false
 
+service:
+  create: false
```

Comment:
@wesuuu Please set to `true` to avoid a breaking-change for all existing users of this chart.